function-app/adb-to-purview/src/Function.Domain/Helpers/PurviewCustomType.cs

```diff
@@ -728,8 +751,10 @@ public async Task<HttpResponseMessage> Send_to_Purview(string payloadJson)
 
             _logger.LogInformation($"Sending this payload to Purview: {payloadJson}");
 
-            var baseurl = $"https://{Environment.GetEnvironmentVariable("PurviewAccountName") ?? ""}.catalog.purview.azure.com/api";
-            var purviewEntityEndpoint = $"{baseurl}/atlas/v2/entity/bulk";
+            var baseurl = $"https://{Environment.GetEnvironmentVariable("PurviewAccountName") ?? ""}.{Environment.GetEnvironmentVariable("ResourceUri") ?? "purview.azure.com"}";
```

Comment:
This breaks anyone's existing appsettings since the current default is https://purview.azure.com